### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: pip-requires.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 smartmin
-Django<1.9
+Django<2.0
 rapidpro-dash
 celery
 coverage
```
Comment (rowanseymour) [inline pip-requires.txt]: you can remove `django-guardian` as well. If you still need the anon user then just copy the migration and util method from rapidpro

File: pip-requires.txt [hunk 2]
```diff
@@ -24,7 +24,7 @@ django-redis
 psycopg2
 xlutils
 mock
-sorl-thumbnail==11.12.1b
+sorl-thumbnail==12.3
 hamlpy
```
Comment (rowanseymour) [inline pip-requires.txt]: this should be replaced by django-hamlpy

File: pip-requires.txt (hunk 1 above)
Comment (rowanseymour) [inline pip-requires.txt]: Can also remove `django-nose` and `nose` if you're not using that for tests - we've not been using it elsewhere for a while now

File: ureport/assets/tests.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 from django.conf import settings
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```
Comment (rowanseymour) [inline ureport/assets/tests.py]: Is this a Django 2.0 change?

Comment (norkans7) [inline ureport/assets/tests.py]: django.core.urlresolvers module is deprecated in 1.10

Comment (rowanseymour) [inline ureport/assets/tests.py]: Ah didn't realise

**Pull request merged**